iceoryx_posh/test/moduletests/test_popo_chunk_queue.cpp

```diff
@@ -42,17 +42,22 @@ class ChunkQueue_testBase
     {
         ChunkManagement* chunkMgmt = static_cast<ChunkManagement*>(chunkMgmtPool.getChunk());
         auto chunk = mempool.getChunk();
-        ChunkHeader* chunkHeader = new (chunk) ChunkHeader();
+        ChunkHeader* chunkHeader = new (chunk) ChunkHeader(chunkMgmtPool.getChunkSize(),
+                                                           PAYLOAD_SIZE,
+                                                           iox::CHUNK_DEFAULT_PAYLOAD_ALIGNMENT,
+                                                           iox::CHUNK_NO_CUSTOM_HEADER_SIZE,
+                                                           iox::CHUNK_NO_CUSTOM_HEADER_ALIGNMENT);
         new (chunkMgmt) ChunkManagement{chunkHeader, &mempool, &chunkMgmtPool};
         return SharedChunk(chunkMgmt);
     }
 
+    static constexpr uint32_t PAYLOAD_SIZE{128};
     static constexpr size_t MEGABYTE = 1 << 20;
     static constexpr size_t MEMORY_SIZE = 4 * MEGABYTE;
     std::unique_ptr<char[]> memory{new char[MEMORY_SIZE]};
     iox::posix::Allocator allocator{memory.get(), MEMORY_SIZE};
-    MemPool mempool{128, 2 * iox::MAX_SUBSCRIBER_QUEUE_CAPACITY, &allocator, &allocator};
-    MemPool chunkMgmtPool{128, 2 * iox::MAX_SUBSCRIBER_QUEUE_CAPACITY, &allocator, &allocator};
+    MemPool mempool{PAYLOAD_SIZE, 2 * iox::MAX_SUBSCRIBER_QUEUE_CAPACITY, &allocator, &allocator};
```

Comment:
U and U

Reply:
congratulations, you killed 6 unicorns :wink:

Reply:
btw. in order to save some unicorns I propose to stick to the first exception of MISRA Rule 10.3
```
An essentially signed integer constant expression, with a rank no greater than signed int, may be assigned to an object of essentially unsigned type if its value can be represented in that type.
```
https://www.misra.org.uk/LinkClick.aspx?fileticket=-BRmxs8omoM%3d&tabid=57